fdbrpc/Locality.cpp

```diff
@@ -185,6 +185,29 @@ ProcessClass::Fitness ProcessClass::machineClassFitness( ClusterRole role ) cons
 			default:
 				return ProcessClass::WorstFit;
 		}
+	case ProcessClass::RateKeeper:
```

Comment:
Your indentation here, and in ProcessClass::DataDistributor, is different than the rest of the ProcessClass cases

Reply:
Aha! You are right!